### PR TITLE
Docker version

### PR DESCRIPTION
**Pull request by munhitsu**

### CODE REVIEW

File: formula-requirements.txt [hunk 1]
```diff
@@ -1 +1 @@
-git@github.com:ministryofjustice/docker-formula.git==v1.0.5
+git@github.com:ministryofjustice/python-formula.git
```
Comment (ashb) [inline formula-requirements.txt]: If this formula is required should it also have an import in an sls somewhere?

Comment (ashb) [inline formula-requirements.txt]: Quite likely missing the import in docker/init.sls:

```
                 ID: docker-py
           Function: pip.installed
        Name: docker-py == 1.0.0
             Result: False
            Comment: State 'pip.installed' found in SLS 'docker' is unavailable
            Started: 
           Duration: 
            Changes:   
```

Comment (munhitsu) [inline formula-requirements.txt]: fixed - long live testing

**Comment by pidah** [comment]
LGTM; tested fine.

**Pull request merged**